src/hyperion/external_interaction/callbacks/xray_centre/ispyb_callback.py

```diff
@@ -75,6 +79,7 @@ def activity_gated_start(self, doc: RunStart):
                 else Store2DGridscanInIspyb(self.ispyb_config, self.params)
             )
             self.ispyb_ids = self.ispyb.begin_deposition()
+        return self._tag_doc(doc)
```

Comment:
Could: Feels like we can just call the parent `start` here and let that do the tagging?